robot_skills/src/robot_skills/world_model_ed.py

```diff
@@ -204,6 +217,33 @@ def get_entity_info(self, uuid):
             rospy.logerr("Cant get entity info of id='{}': {}".format(id, e))
             return GetEntityInfoResponse()
 
+    def get_map(
+        self, uuids: List[str], background: str = "white", print_labels: bool = True, width: int = 0, height: int = 0
+    ) -> Tuple[np.ndarray, FrameStamped, float, float]:
+        """
+        :param uuids: Entities that should be in view in the generated map
+        :param background: Background color of the map
+        :param print_labels: Should entity labels be printed
+        :param width: image width (default: 0, defaults to 1024)
+        :param height: image height (default: 0, defaults to 600)
+        :returns: The generated map, the position of the top-left corner of the image and the pixels/meter in both axis
+        """
+        req = MapRequest()
+        req.entities_in_view = uuids
+        req.background = req.WHITE  # default
+        if hasattr(req, background.upper()):
+            req.background = getattr(req, background.upper())
```

Comment:
Give a notification if a non-valid color was requested. Rather than ignoring it.